packages/@react-spectrum/dnd/docs/dragAndDrop.mdx

```diff
@@ -0,0 +1,256 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import ChevronRight from '@spectrum-icons/workflow/ChevronRight';
+import docs from 'docs:@react-spectrum/dnd';
+import dndDocs from 'docs:@react-types/shared/src/dnd.d.ts';
+import {FunctionAPI, PageDescription, TypeContext, InterfaceType} from '@react-spectrum/docs';
+import {Keyboard} from '@react-spectrum/text';
+
+---
+category: Concepts
+keywords: [drag and drop, dnd]
+after_version: 3.0.0
+---
+
+# Drag and Drop
+
+This page describes how to enable drag and drop functionality for the various React Spectrum components that support it.
+
+## Introduction
+
+Drag and drop is an operation performed when a user "grabs" a UI element via their pointing device and drops it on a different location or element.
+It has become a common interaction in many modern applications due to its flexibility across a variety of use cases but can come saddled with
+many accessibility issues, including but not limited to keyboard interaction parity and difficulties mimicking drag operations with mouse emulators.
+React Spectrum components that support drag and drop operations seek to address these issues by providing full keyboard and screen reader drag and drop
+support.
+
+If you are interested in adding drag and drop support to your own components, check out the [drag and drop react-aria hooks](https://www.npmjs.com/package/@react-aria/dnd)!
+
+## API
+
+<FunctionAPI function={docs.exports.useDragHooks} links={docs.links} />
+<FunctionAPI function={docs.exports.useDropHooks} links={docs.links} />
+
+## Usage
```

Comment:
I think the API and Usage sections here could be combined, with the function API right above the props table for each.

Reply:
Just wanna note that the current structure mimics the aria hook docs structure. I'm fine with combining the sections (I agree that it would be clearer), but lemme know if we should remain consistent w/ the hooks doc structure instead

Reply:
I feel this is different since this page is more conceptual